src/Administration/Resources/administration/src/module/sw-customer/page/sw-customer-detail/index.js

```diff
@@ -113,13 +113,17 @@ Component.register('sw-customer-detail', {
     },
 
     methods: {
+        defaultCriteria() {
+            return new Criteria();
+        },
+
         createdComponent() {
             this.isLoading = true;
             if (this.$route.params.id) {
                 this.customerId = this.$route.params.id;
 
                 if (!this.createMode) {
-                    const criteria = new Criteria();
+                    const criteria = this.defaultCriteria();
                     criteria.addAssociationPaths([
```

Comment:
This AssociationPaths should be added in the `defaultCriteria()` function
then the get call can simply be 
`this.customerRepository.get(this.customerId, this.context, this.defaultCriteria())`